### DISCUSSION TITLE
Issue with RFC9204?

### DISCUSSION THREAD

**Discussion by IsaJafarov** [ietf-tools/datatracker#8044] [General]
Here is a quote from RFC 9204 (QPACK):
    - An encoder stream is a unidirectional stream of type 0x02. It carries an unframed sequence of encoder instructions from encoder to decoder.
    - A decoder stream is a unidirectional stream of type 0x03. It carries an unframed sequence of decoder instructions from decoder to encoder.

In RFC 9000 (QUIC), we see that stream type 0x02 is for Unidirectional Client-Initiated streams and 0x03 is for Unidirectional Server-Initiated streams. Based on this information, we can conclude that an encoder stream can only be created by the client and a decoder stream can only be created by the server. However, it contradicts the fact that each peer can open both encoder and decoder streams. Even in practice, we can see servers opening encoder streams on stream #7 and clients opening decoder streams on stream #10.

Is there something wrong with the stream id?

**Comment by richsalz** [comment]
This GitHub repo is for the DataTracker tool used by the IETF.  Your question about QUIC RFC's should be posted to the working group mailing list, quic@ietf.org

**Reply by IsaJafarov** [reply]
Thank you for letting me know.
